source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -62,12 +62,16 @@ Episode 129 of the EmberWeekend podcast featured [@mansona](https://github.com/m
 
 [@mansona](https://github.com/mansona) also introduced Guidemaker:  A fully functional, static site implementation of a documentation site and EmberJS with fully out of the box SEO friendly output! With support to be hosted on S3 or any other site hosting solution. Check out the [github](https://github.com/empress/guidemaker) repo here.
 
-Listen to the full [podcast]( https://emberweekend.com/episodes/empress-the-ember-press). 
+Listen to the full [podcast]( https://emberweekend.com/episodes/empress-the-ember-press).
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Singleton Record Data RFC](https://github.com/emberjs/rfcs/pull/461) 💾
+Want to learn more about how some of Ember Data's API's are developing? Take a peek at the Singleton Record Data RFC recently opened by [@runspired](https://github.com/runspired).
```

Comment:
I _think_ it'd be `APIs` since it's not a genitive (possessive) case, but I'm not a native speaker in contrast to you - let me know your thoughts!

```suggestion
Want to learn more about how some of Ember Data's APIs are developing? Take a peek at the Singleton Record Data RFC recently opened by [@runspired](https://github.com/runspired).
```

Reply:
Seems good! I think you're correct about this. Thank you for the review!